tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+envlist =
+  py27
+  lint
+skipsdist = True
```

Comment:
The `skipsdist` tells tox to not try to install the collectors as a Python package (i.e. `python setup.py install`), as it isn't a Python package at all. Without this, it would fail complaining that there's no `setup.py`.

This is also the reason why I've added `-r{toxinidir}/requirements.txt` in the dependencies. If it were a package, Python would install its dependencies when installing it. As it's not, we have to install them manually.